server/portal/apps/projects/workspace_operations/shared_workspace_migration.py

```diff
@@ -42,19 +44,19 @@ def migrate_project(project_id):
     try:
         owner = v2_project.owner.username
     except AttributeError:
-        pass
-    try:
-        owner = v2_project.pi.username
-    except AttributeError:
-        print('No owner or PI specified')
-        return
+        try:
+            owner = v2_project.pi.username
+        except AttributeError:
+            print('No owner or PI specified')
+            return
 
     try:
         create_workspace_system(client, project_id, v2_project.title, v2_project.description)
     except BaseTapyException as e:
         if 'SYSAPI_SYS_EXISTS' in e.message:
             print('A Tapis V3 workspace already exists for this system.')
             return
+        raise
```

Comment:
Do we want this utility to raise exceptions? It might be better to print the error so we can come back and debug the offending workspace separately, instead of requiring the full migration to be restarted.

Reply:
Good call, updated.